### ISSUE TITLE
Document type declarations in SVG content documents: bare minimum

### ISSUE THREAD

**Issue by murata2makoto** [w3c/epub-specs#1358]
I understand that some SVG editors create this document type declaration:

```
<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" 
  "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
```

Do people want to allow something else?  Different public identifiers are obviously bad, since they mean "I'm not SVG".  The absence of system identifiers is fine.  But does somebody want to allow different system identifiers?  A sensible reason is to declare internal or external parsed entities in your external DTD subset.  But such an external DTD subset should not have  "-//W3C//DTD SVG 1.1//EN"  as the public identifier.

If we allow the above and 

```
<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN">
```

and nothing else, we do not have to worry about possible definitions of parsed entities in external DTD subsets and external parameter entities.

**Comment by murata2makoto** [comment]
I forgot to mention internal DTD subsets.  They should be allowed since they have been allowed by EPUB3,

**Comment by iherman** [comment]
I believe the only DTD-s that we should allow are the one [in the SVG specification](https://www.w3.org/TR/SVG11/conform.html) (the one you use in your starting example).

**Comment by murata2makoto** [comment]
@iherman 

Both the system identifier and the public identifier?

**Comment by iherman** [comment]
To be very pragmatic: whatever makes the authors', checkers', and RS implementers' life easier; the goal is to ignore those anyway. The SVG specification refers to 

```
<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" 
  "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
```

in all its examples, for example, I bet this is the DTD string all tools use. I do not know whether any of the other forms that you refer to has any practical value.

**Comment by murata2makoto** [comment]
As shown in #1354, we have allowed internal subsets and do not have any reasons to disallow them.  Let me make a proposal.

- A document type declaration [XML] in an SVG content document MUST NOT specify external identifiers except when it specifies "-//W3C//DTD SVG 1.1//EN" as a public identifier and also specifies "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd" as a system identifier.  Note: This restriction does not allow declarations of external parsed entities or external parameter entities in the internal DTD subset of an SVG content document.

**Comment by iherman** [comment]
This issue was discussed in [a meeting](https://www.w3.org/publishing/groups/epub-wg/Meetings/Minutes/2020-11-06-epub#section3).

- `RESOLVED:  Merge PR #1368 to address outstanding DTD issues, and close GH issues 1369-1373`
<details><summary><i class="differentiate">View the transcript</i></summary>
Wendy Reid: we had resolutions at the F2F, and further discussions on github<br/>
… and came to a happy place<br/>
Matt Garrish: <a href="https://github.com/w3c/epub-specs/pull/1368">https://github.com/w3c/epub-specs/pull/1368</a><br/>
Matt Garrish: where we ended up was…<br/>
… we put in an allowance for a specific set of external identifiers that we have put in an appendix<br/>
… we have SVG and MathML that are allowed to be used in content docs or in separate files<br/>
… and we made a restriction against external entities in the internal DTD subset<br/>
… so it prevents some security issues but eases authoring<br/>
… so we’ll no longer force people to remove SVG DTDs from tool-generated files<br/>
… I’m hoping this is it :)<br/>
Ivan Herman: tech comment<br/>
… in fact, the changes are such that<br/>
… makes possible something that I’m not sure we really use<br/>
… I can define as part of an internal entity something that won’t go out to the network<br/>
… I’m not sure if this feature is in use<br/>
… formal comment<br/>
… there was a formal resolution on the previous version; this PR slightly changes that<br/>
… can we get a formal resolution to merge, and also close a bunch of issues which were examples of the problem?<br/>
Proposed resolution: Merge PR #1368 to address outstanding DTD issues, and close GH issues 1369-1373 (Wendy Reid)<br/>
Garth Conboy: +1<br/>
Matt Garrish: +1<br/>
Ivan Herman: +1<br/>
Charles LaPierre: +1<br/>
Matthew Chan: +1<br/>
Wendy Reid: +1<br/>
Brady Duga: +1<br/>
George Kerscher: +1<br/>
Laura Brady: +1<br/>
Bill Kasdorf: +1<br/>
Ben Schroeter: +1<br/>
Resolution #1: Merge PR #1368 to address outstanding DTD issues, and close GH issues 1369-1373<br/>
</details>

**Comment by iherman** [comment]
@wareid I believe this issue can also be closed
